### PR TITLE
Add some more intro text, clarify the justification

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: docs/worker-node/install-cvmfs.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-By installing CVMFS, you will be able to support a much larger range of jobs on your site.
-OASIS uses CVMFS to provides easy access to worker node software, CA certificates, CRLs, etc. via CVMFS.
-It also provides easy access to StashCache resources.
+- The [worker node client](/worker-node/install-wn-oasis)
+- CA and VO security data
+- Software used by VOs
+- Data stored in [StashCache](/data/stashcache/overview).
+
+Use this page to learn how to install, configure, run, test, and troubleshoot the CVMFS client from the OSG software
+repositories.
```
Comment (DrDaveD) [inline docs/worker-node/install-cvmfs.md]: I find this wording confusing, since "from the OSG software repositories" really only applies to the install step.  I would drop that phrase or reword it to apply to only install.

Comment (brianhlin) [reply docs/worker-node/install-cvmfs.md]: I think the other items apply to "from the OSG software repositories". We're telling people how to set up and use the OSG-packaged CVMFS and make no claims about these instructions applying to say EGI's CVMFS.
